pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
+
+// These are the valid service statuses of pods.
+const (
+	PodInService = "InService"
```

Comment:
On Sat, Feb 7, 2015 at 3:04 PM, Mike Danese <notifications@github.com>
wrote:

> NodeConditionStatus will be so similar to PodConditionStatus that
> NodeConditionStatus could be renamed to ConditionStatus and used for both
> cases, and eventually maybe for ContainerCondition.
>
Works for me.